src/utils/arfs_builders/arfs_folder_builders.ts

```diff
@@ -4,13 +4,25 @@ import { ArFSPrivateFolder, ArFSPublicFolder } from '../../arfs_entities';
 import { ArweaveAddress } from '../../types/arweave_address';
 import { CipherIV, DriveKey, FolderID } from '../../types';
 import { ArFSFileOrFolderBuilder } from './arfs_builders';
+import { EID, EntityID } from '../../types/entity_id';
+import { stubEntityID } from '../stubs';
+
+export const ROOT_FOLDER_ID_PLACEHOLDER = 'root folder';
+
+// A utility type to assist with fail-safe decryption of private entities
```

Comment:
This would be a utility type that creates a placeholder for the parentFolderId field when its missing; which presumably means this folder is the root folder of the drive.

Reply:
yea lemme update this

Reply:
8bca2e9